### DISCUSSION TITLE
Can't enable v7 feature

### DISCUSSION THREAD

**Discussion by iron3oxide** [uuid-rs/uuid#695] [Q&A] [answered]
Use case:
I want to (automatically) generate a v7 UUID as the primary key for a DB table. I'm using `Uuid` via `sqlx::types`, but using it on its own didn't change anything.

Problem:
Neither `Uuid::new_v7()` nor `Uuid::now_v7()` can be found, even though `RUSTFLAGS="--cfg uuid_unstable"` is set and both the `std` and `v7` features are enabled.

What I already tried:
Removing and readding the `uuid` crate after resetting the environment variable

Using 1.4.0, any help would be appreciated

**Comment by KodrAus** [comment, marked as answer]
cc #696 Thanks for the report @iron3oxide. I haven't been able to replicate this yet, but it seems like something people are running into.

**Reply by iron3oxide** [reply]
Just got it fixed the same way the person who reported the issue did :facepalm:
